### DISCUSSION TITLE
Kibana menu customization

### DISCUSSION THREAD

**Discussion by strf0x1** [Security-Onion-Solutions/securityonion#1574] [Unsupported Versions] [answered]
Just want to say this new release is the greatest thing. So grateful for this project. 

What's the best way to go about enabling components in the Kibana view such as Index Management, and Rollup Jobs? Stuff that is available in a default Kibana install. Thanks for any help.

**Comment by weslambert** [comment, marked as answer]
I believe you are referring to `Features`.  If you would like to enable features, please have a look at `so-features-enable`.  I've created an issue to update the documentation, here: https://github.com/Security-Onion-Solutions/securityonion/issues/1652

**Reply by strf0x1** [reply]
Thank you!
